.golangci.yml

```diff
@@ -132,3 +132,6 @@ issues:
     # We want to show examples with http.Get
     - linters: [noctx]
       path: internal/memhttp/memhttp_test.go
+    # We need to initialize a map of all protocol headers
+    - linters: [gochecknoglobals]
+      path: header.go
```

Comment:
For a single violation, seems _way_ better to put this at the actual point of use via `//nolint:gochecknoglobals`.

Reply:
Will create a PR to convert over the others too, current convention is to add them here.